oktapam/resource_project_test.go

```diff
@@ -39,7 +39,7 @@ func TestAccProject(t *testing.T) {
 		RDPSessionRecording:    utils.AsBoolPtrZero(true, true),
 		SSHSessionRecording:    utils.AsBoolPtrZero(true, true),
 		GatewaySelector:        utils.AsStringPtr("env=test"),
-		SSHCertificateType:     utils.AsStringPtr("CERT_TYPE_ED25519_01"),
```

Comment:
Any reason the type was changed?

Reply:
I think if I'm reading the test case right, L42 needs to match L85 so that the test case passes. I picked something other than `CERT_TYPE_ED25519_01` so that the test would actually check that the value was updated. I can definitely pick something other than RSA_01 though.

Changing it back results in a failure when running the acceptance tests:
```log
TESTARGS="-run TestAccProject" make testacc
TF_ACC=1 go test ./... -v -run TestAccProject -timeout 120m   
?       github.com/okta/terraform-provider-oktapam      [no test files]
=== RUN   TestAccProjectGroup
--- PASS: TestAccProjectGroup (13.34s)
=== RUN   TestAccProject
    resource_project_test.go:45: Step 2/3 error: Check failed: 1 error occurred:
                * Check 1/6 error: expected project does not match returned project.
         {
          Name: "test_acc_project_MGTNLLDX1Q",
          ID: "1ab087a9-7f9c-4dcf-901d-446a3e048792",
          Team: "wojoinc",
          DeletedAt: nil,
          NextUnixGID: 63400,
          NextUnixUID: 61200,
          CreateServerUsers: true,
          ForwardTraffic: true,
          RDPSessionRecording: true,
          RequirePreAuthForCreds: false,
          SSHSessionRecording: true,
          GatewaySelector: "env=test",
        - SSHCertificateType: "CERT_TYPE_RSA_01",
        + SSHCertificateType: "CERT_TYPE_ED25519_01",
          UserOnDemandPeriod: 10,
         }
        
--- FAIL: TestAccProject (9.14s)
```

Reply:
That reason is fine, just curious.

Reply:
Okay sounds good!